README.md

```diff
@@ -30,14 +30,14 @@ the relevant OpenFeature SDK dependency manually.
 <!---x-release-please-start-version-->
 In the dependencies section of Package.swift add:
 ```swift
-.package(url: "git@github.com:spotify/confidence-openfeature-provider-swift.git", from: "0.1.4")
+.package(url: "git@github.com:spotify/confidence-sdk-swift.git", from: "0.1.4")
```

Comment:
not a super strong opinion but should we decide between `-swift` and `-ios`?

Reply:
Also not sure, but I imagine if we ever were to support macOS or other Apple OSes, the code would still live in this repo